ocaml/typing/ctype.ml

```diff
@@ -542,7 +543,7 @@ let really_closed = ref None
      are expanded to check whether the apparently-free variable would vanish
      during expansion.
    - We collect both type variables and row variables, paired with a boolean
-     that is [true] if we have a row variable.
+     that is [false] if we have a row variable.
```

Comment:
!  This is alarming

Reply:
:alarm_clock: 

Yes, it is. I was alarmed, too. Which is why I corrected it. If you wish, you can double-check that my fix is an understanding of the code. The good news is both the consumer and producer of this information were on the same page -- it's just the comment that was wrong.